### DISCUSSION TITLE
Organization

### DISCUSSION THREAD

**Discussion by donaldli2020** [ProdigyErrorCodes/ProdigyErrorCodes#15] [Votes]
Should we make an organization?
I will press 👍 and 👎 for convenience.

**Comment by afkvido** [comment]
> Should we make an organization?
> I will press 👍 and 👎 for convenience.

I think that's a good idea. What should we name it?

**Reply by donaldli2020** [reply]
idk

**Comment by donaldli2020** [comment]
@afkvido I invited you to an organization @ProdigyErrorCodes
